apps/remix-ide/src/app/tabs/locales/en/filePanel.json

```diff
@@ -127,5 +127,6 @@
   "filePanel.movingFileFailedMsg": "Unexpected error while moving file: {src}",
   "filePanel.movingFolderFailed": "Moving Folder Failed",
   "filePanel.movingFolderFailedMsg": "Unexpected error while moving folder: {src}",
-  "filePanel.workspaceActions": "Workspace actions"
+  "filePanel.workspaceActions": "Workspace actions",
+  "filePanel.saveCodeSample": "This code sample will not be persisted. Please click here to save this code as a persisted workspace."
```

Comment:
This code-sample workspace will not be persisted. Click here & rename to save it.
or
This is a temporary workspace. Click here & rename to save it.